src/CustomiesListener.php

```diff
@@ -36,16 +34,10 @@ public function __construct() {
 
 	public function onDataPacketSend(DataPacketSendEvent $event): void {
 		foreach($event->getPackets() as $packet){
-			if($packet instanceof BiomeDefinitionListPacket) {
-				// ItemComponentPacket needs to be sent after the BiomeDefinitionListPacket.
-				if($this->cachedItemComponentPacket === null) {
-					// Wait for the data to be needed before it is actually cached. Allows for all blocks and items to be
-					// registered before they are cached for the rest of the runtime.
-					$this->cachedItemComponentPacket = ItemComponentPacket::create(CustomiesItemFactory::getInstance()->getItemComponentEntries());
-				}
-				foreach($event->getTargets() as $session){
-					$session->sendDataPacket($this->cachedItemComponentPacket);
-				}
+			if($packet instanceof ItemRegistryPacket) {
+				(function() : void{
+					$this->entries = array_merge($this->entries, CustomiesItemFactory::getInstance()->getItemTableEntries());
```

Comment:
$this->entries doesnt exist

Reply:
It is a yet another way to hack private property without using ReflectionClass.